src/components/panel/ingen-gjeldende-vedtak/ingen-gjeldende-vedtak-panel.less

```diff
@@ -0,0 +1,25 @@
+@import "~nav-frontend-core/less/_variabler";
+
```

Comment:
Ikke et must, men siden dette er duplisert fra ingen-tidligere-vedtak-panel så kunne vi laget en felles komponent for dette